libs/ledger-live-common/src/__tests__/test-helpers/bridge.ts

```diff
@@ -542,6 +542,10 @@ export function testBridge<T extends TransactionCommon>(data: DatasetTest<T>): v
               recipient: account.freshAddress,
             });
           });
+          /*
+            FIXME: Flaky test
+            ISSUE: Sometimes the "expireTimestamp" is off by 1 microsecond
+          */
```

Comment:
was it flaky before with `expect(t2).toBe(t3);` ?

Reply:
I'm not sure. This test fails 1 on every 10 runs, more or less.
I don't know for sure if it was already flaky, or if the issue is on our side tbh.

Reply:
Fixed. I changed the `expireTimestamp` to be in milliseconds instead of microseconds.